ssl/ssl_rsa.c

```diff
@@ -904,7 +920,7 @@ int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file)
     int ret = 0;
     BIO *bin = NULL;
     size_t num_extensions = 0, contextoff = 0;
-    unsigned int version = 0;
+    unsigned int version;
```

Comment:
It's true that we're not consistent...  as a matter of fact, we were consistently putting all variables at the top of functions, which means you sometimes have to scroll for miles before you find where it's used.  I prefer to limit the scope of variables when possible...  and I guess I'm trying to shove the rest of you that way as well ;-)